### PR TITLE
Customization of the template content closes #3

### LINKED ISSUE #3: Customize template content
This repo has been created with the template. Customize the repo to retoch (with a module `retorch-annotations`, namespace `giis.retorch.annotations`). The result must be a fully configured project, but without unnecessary files/configs and without any source of annotations (this is for another PR).

### PR DESCRIPTION
**Pull request by augustocristian**
- Removed modules not used
- Changed module-project names to retorch and retorch-annotations
- Namespace renamed to giis.retorch.annotations
- Test case modifed to pass (uses one of the modules removed)
- Some little spell mistakes corrected.
- Added .settings to the module retorch-annotations (ignored originally by the gitignore)
- SonarQube configurations performed 
- Customization of the master and retorch-annotations pom.xml files
- First version of the README.md with basic information about the annotations, contributing, citation,contact etc.

### CODE REVIEW

File: .idea/.gitignore
Comment (javiertuya) [inline .idea/.gitignore:1]: Remove the idea configuration files to avoid polluting the repository, and add to .gitignore

File: pom.xml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-	<artifactId>samples-giis-template</artifactId>
+	<artifactId>retorch</artifactId>
 	<version>1.0.1-SNAPSHOT</version>
 	<packaging>pom</packaging>
 
-	<name>samples-giis-template</name>
-	<description>Multimodule maven project template</description>
+	<name>retorch</name>
+	<description>Resource-aware Orchestration of End-to-End Test Cases</description>
 	<url>http://github.com/giis-uniovi/samples-giis-template</url>
```
Comment (javiertuya) [inline pom.xml:10]: Customize url

File: retorch-annotations/pom.xml [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
 		<version>1.0.1-SNAPSHOT</version>
 	</parent>
-	<artifactId>samples-giis-template-m1</artifactId>
+	<artifactId>retorch-annotations</artifactId>
 	<packaging>jar</packaging>
-	<name>samples-giis-template-m1</name>
-	<description>Multimodule maven project template, module without dependencies</description>
+	<name>retorch-annotations</name>
+	<description>Resource-aware Orchestration of End-to-End Test Cases, module with annotations</description>
 	<url>http://github.com/giis-uniovi/samples-giis-template</url>
```
Comment (javiertuya) [inline retorch-annotations/pom.xml:12]: Customize this url

File: sonar-project.properties [hunk 3]
```diff
@@ -1,9 +1,9 @@
 sonar.projectKey=my:samples-giis-template
```
Comment (javiertuya) [inline sonar-project.properties:1]: key should be my:retorch

However, do we need sonarqube for the annotations?. I think it is no necessary. If you agree, please, remove this file and the corresponding job at `.github/workflows`. If you want SonarQube now, update the key and create an issue to me to configure sonar

**Review by javiertuya — CHANGES_REQUESTED** [review]

**Comment by javiertuya** [comment]
@augustocristian Just a tip: you don't need to always push every internal commit that you do because each push starts all workflows and sonar (if configured)

**Comment by augustocristian** [comment]
Dear @javiertuya 
Now everything seems to be working correctly. I made changes to the URL artifacts in the pom.xml. Initially, I thought this information/parametrization was included in the Actions file. However, after reviewing it, I realized it needed to be modified in the pom.xml instead.

File: README.md
[the remark this answers is not in the document]
Comment (javiertuya) [reply README.md:1]: Customize this description (README.md)

**Review by javiertuya — CHANGES_REQUESTED** [review]
FYI You don't need use mentins to request review. you vsn use the button at the top right of the screen to request review

File: README.md [hunk 4]
```diff
@@ -2,14 +2,128 @@
...
+    <a href="https://github.com/giis-uniovi/retorch"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/giis-uniovi/retorch">View Demo</a>
+    ·
+    <a href="https://github.com/giis-uniovi/retorch/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/giis-uniovi/retorch/issues">Request Feature</a>
+  </p>
+</div>
```
Comment (javiertuya) [inline README.md]: The readme is a markdown document. Therefore, html should be exclusively used to things that we can't do in markdown. I I'm not wrong all html here can be written using markdown. The most difficult thing could be the TOC, you can see an example at https://github.com/giis-uniovi/qacover

File: README.md [hunk 5]
```diff
@@ -2,14 +2,128 @@
...
+
+```java
+    @Resource(resID = "LoginService", replaceable = {})
+@AccessMode(resID = "LoginService", concurrency = 10, sharing = true, accessMode = "READONLY")
+@Resource(resID = "OpenVidu", replaceable = {"OpenViduMock"})
+@AccessMode(resID = "OpenVidu", concurrency = 10, sharing = true, accessMode = "NOACCESS")
+@Resource(resID = "Course", replaceable = {"Forum"})
+@AccessMode(resID = "Course", concurrency = 10, sharing = true, accessMode = "READONLY")
+@ParameterizedTest
+@MethodSource("data")
```
Comment (javiertuya) [inline README.md:88]: Check indentation, this is literally displayed

File: README.md [hunk 6]
```diff
@@ -2,14 +2,128 @@
...
+            driver=user.getDriver();
+            this.slowLogin(user,usermail,password);//24 lines
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Contributing
+
+See the general contribution policies and guidelines for *giis-uniovi* at
 [CONTRIBUTING.md](https://github.com/giis-uniovi/.github/blob/main/profile/CONTRIBUTING.md).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (javiertuya) [inline README.md:102]: As before, remove all html, check the whole document

File: README.md [hunk 7]
```diff
@@ -2,14 +2,128 @@
...
+https://doi.org/10.1007/s11219-020-09505-2
+```
+[Full Article available](https://link.springer.com/article/10.1007/s11219-020-09505-2) - [Authors version](https://digibuo.uniovi.es/dspace/bitstream/handle/10651/55405/RETORCHSQJExtension_BUO.pdf;jsessionid=0E661594C8732B8D2CA53636A31E4FD5?sequence=1) - 
+[Download citation](https://citation-needed.springer.com/v2/references/10.1007/s11219-020-09505-2?format=refman&flavour=citation)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Acknowledgments
+
+This work has been developed under the TestBUS (PID2019-105455GB-C32) and EQUAVEL (PID2022-137646OB-C32) projects supported by [Ministry of Science and Innovation (SPAIN)](https://www.ciencia.gob.es/)
```
Comment (javiertuya) [inline README.md:128]: Remove equavel at the moment, it is not officially approved

File: README.md [hunk 8]
```diff
@@ -2,14 +2,128 @@
...
+attributes:
+
+- resID: unique resource identifier
+- replaceable: list of resources that can replace the current ones
+
+```java
+    @Resource(resID = "LoginService", replaceable = {}) 
+```
+
+The following code snippets illustrates a tes case annotated with several resources and access modes:
```
Comment (javiertuya) [inline README.md:78]: spelling: tes case

File: README.md [hunk 9]
```diff
@@ -2,14 +2,128 @@
...
+<div align="center">
+  <a href="https://giis.uniovi.es/?lang=en">
+    <img src="https://giis.uniovi.es/icons/giis-color-medium.gif" alt="Logo" width="110" height="75">
+  </a>
 
-See the general contribution policies and guidelines for *giis-uniovi* at 
+<h3 align="center">RETORCH: Resource-aware End-to-End Test Orchestration</h3>
+
+  <p align="center">
+    This repository contains a series of components that compose RETORCH, a E2E test orchestration rchestration framework which aims
```
Comment (javiertuya) [inline README.md:17]: spelling: test orchestration rchestration

File: README.md [hunk 10]
```diff
@@ -2,14 +2,128 @@
...
+See the general contribution policies and guidelines for *giis-uniovi* at
 [CONTRIBUTING.md](https://github.com/giis-uniovi/.github/blob/main/profile/CONTRIBUTING.md).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Contact
+
+Cristian Augusto - [Contact me!](mailto:augustocristian@uniovi.es)
+
+Project
+Link: [https://github.com/giis-uniovi/retorch](https://github.com/giis-uniovi/retorch)
```
Comment (javiertuya) [inline README.md:109]: redundant, this is this repo

File: README.md [hunk 11]
```diff
@@ -2,14 +2,128 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Citing this work
+RETORCH test orchestration framework:
+```
+Cristian Augusto, Jesús Morán, Antonia Bertolino, Claudio de la Riva, and Javier Tuya, 
+“RETORCH: an approach for resource-aware orchestration of end-to-end test cases,” 
+Software Quality Journal, vol. 28, no. 3, 2020.
+https://doi.org/10.1007/s11219-020-09505-2
+```
```
Comment (javiertuya) [inline README.md:120]: This reference should be formatted like a paper reference, not as source code

**Review by javiertuya — CHANGES_REQUESTED** [review]

**Comment by augustocristian** [comment]
Personalized the README, following the sections of td-rules plus some extra (contact  and acks) as well as the the first draft of the annotations

**Comment by javiertuya** [comment]
@augustocristian Remember that the comment that originated this was *Customize this description: a simple statement to say what is retorch, the annotations. Include a note to explain that annotations are the only available now, but in a near future, other retorch componentes will be available.".

You can keep the details, but please, include the note referred to in the second part

File: README.md (hunk 4 above)
Comment (augustocristian) [reply README.md]: Thanks Javier! Working on it

File: README.md (hunk 11 above)
Comment (augustocristian) [reply README.md:103]: @javiertuya , can be interesting study includying this type of files? https://citation-file-format.github.io/

**Comment by augustocristian** [comment]
> 

I apologize for the misunderstanding. I understood that this minor personalization was intended only in the pom.xml file (with the note that not all components are available at the moment), so when this morning I realized the change resquest in the readme, I thought it was to include all (at least a preliminary version)

File: README.md (hunk 11 above)
Comment (javiertuya) [reply README.md]: Sounds good, but not for now, we can't delay any longer this PR

Comment (augustocristian) [reply README.md]: Would you like me to open an issue in this project or the template project with the provided links and information? @javiertuya
Perhaps we can address it in the near to medium future for all projects.

**Comment by javiertuya** [comment]
@augustocristian Is this for review? there are 2 unresolved conversations. No PR can be merged if any conversation is unresolved.

File: README.md [hunk 12]
```diff
@@ -2,14 +2,113 @@
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=my%3Aretorch&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=my%3Aretorch)
 [![Maven Central (annotations)](https://img.shields.io/maven-central/v/io.github.giis-uniovi/retorch-annotations)](https://central.sonatype.com/artifact/io.github.giis-uniovi/retorch-annotations)
 
-# samples-giis-template
+<h1 align="center"> RETORCH: Resource-aware End-to-End Test Orchestration</h1>
+<div align="center">
+ <a href="https://giis.uniovi.es/?lang=en">
+    <img src="https://giis.uniovi.es/icons/giis-color-medium.gif" alt="Logo" width="110" height="75">
+</a >
+</div>
```
Comment (javiertuya) [inline README.md:10]: Still html

File: README.md [hunk 13]
```diff
@@ -2,14 +2,113 @@
...
+    - [Contributing](#contributing)
+    - [Contact](#contact)
+    - [Citing this work](#citing-this-work)
+    - [Acknowledgments](#acknowledgments)
+
+## Quick-start
+
+[TO-DO]
+
+[(Back to the top)](#contents)
```
Comment (javiertuya) [inline README.md:36]: Remove back to the top, this file is small enough to do not need more complications

File: README.md [hunk 14]
```diff
@@ -2,14 +2,113 @@
...
+
+[(back to the top)](#contents)
+
+## Citing this work
+
+RETORCH E2E Test Orchestration framework:
+
+- Cristian Augusto, Jesús Morán, Antonia Bertolino, Claudio de la Riva, and Javier Tuya,
+  “RETORCH: an approach for resource-aware orchestration of end-to-end test cases,”
+  Software Quality Journal, vol. 28, no. 3, 2020.
```
Comment (javiertuya) [inline README.md:103]: journal in italics

**Review by javiertuya — APPROVED** [review]

**Pull request merged**